Bus Scheduler/src/components/tripSearch/store/hook.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect } from 'react';
+import { GET } from '../../../store/api/axios'
+import { tripListApiUrl } from '../../../store/api/urls'
+
+import { useCustomContext } from '../../../store/contextProvider';
+import { TripSearchContext } from './tripSearchProvider';
+
+export const useFetchTrips = () => {
```

Comment:
custom hooks can be /components/component-name/hooks

Reply:
Done